src/ns/adapters/outbound/smtp_client.py

```diff
@@ -51,6 +51,13 @@ class SmtpClientConfig(BaseSettings):
     use_starttls: bool = Field(
         default=True, description="Boolean flag indicating the use of STARTTLS"
     )
+    smtp_timeout: PositiveInt | None = Field(
```

Comment:
To be type consistent with what the `SMTP` class expects this should be `PositiveFloat` instead

Reply:
Yes, that is the way it should be. Look here: [7991526](https://github.com/ghga-de/notification-service/pull/20/commits/7991526a5433694407a0891eff2eda7ee3ca6b10)